WPFHospitalEditor/HospitalMap.xaml.cs

```diff
@@ -140,6 +139,50 @@ public void Medication_Search(object sender, RoutedEventArgs e)
             }
         }
 
+        public void appointmentSearch_Click(object sender, RoutedEventArgs e)
+        {
+            clearAllResults();
+            if (InvalidInputForAppointment())
+            {
+                MessageBox.Show("Invalid input.");
+            }
+            else
+            {   
+                int index = doctorsComboBox.SelectedIndex;
+                DateTime startDate = DateTime.ParseExact(startDatePicker.Text + AllConstants.DayStart, "MM/dd/yyyy HH:mm", null);
+                DateTime endDate = DateTime.ParseExact(endDatePicker.Text + AllConstants.DayEnd, "MM/dd/yyyy HH:mm", null);
+                RecommendationRequestDto recommendationRequestDto = new RecommendationRequestDto()
+                {
+                    DoctorId = doctorServerController.GetDoctorsByDepartment(regularExaminationDepartment).ElementAt(index).DoctorId,
+                    SpecialtyId = regularExaminationDepartment,
+                    TimeInterval = new TimeInterval(startDate, endDate),
+                    Preference = GetRecommendationPreference()
+                };
+                if (schedulingController.GetAppointments(recommendationRequestDto) != null)
+                {
+                    appointemntSearchResult = schedulingController.GetAppointments(recommendationRequestDto);
+                    SearchResultDialog appointmentDialog = new SearchResultDialog(this, SearchType.AppointmentSearch);
+                    appointmentDialog.ShowDialog();
+                }
+                else
+                {
+                    MessageBox.Show("Nema slobodnih termina za traženi period!");
+                }
```

Comment:
This text is on serbian, i guess we need to use english everywhere. Do you even need messageboxes?

Reply:
I need it because there can be no search results. Changed to english